Azure.ResourceManager.Core/SubscriptionOperations.cs

```diff
@@ -48,15 +48,13 @@ internal SubscriptionOperations(AzureResourceManagerClientOptions options, Resou
         {
         }
 
+        protected override ResourceType ValidResourceType => ResourceType;
+
         /// <summary>
         /// Gets the subscription client.
         /// </summary>
-        internal SubscriptionsOperations SubscriptionsClient => GetClient((uri, cred) =>
-            new ResourcesManagementClient(
-                uri,
-                Guid.NewGuid().ToString(),
-                cred,
-                ClientOptions.Convert<ResourcesManagementClientOptions>())).Subscriptions;
+        internal SubscriptionsOperations SubscriptionsClient => GetClient<ResourcesManagementClient>((uri, cred) =>
```

Comment:
Resharper actually shows the GetClient<Type> is redundant. Thus I removed it. Also multiline is earlier to read.